### ISSUE TITLE
$unused++ is not detected

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#336]
The "unused variable" inspection consider `$unused++` inside a `for()` as used, even if it never is really used.

```php
$unusedInForeach = 0;

foreach ([] as $item) {
    // Increased, but never used.
    $unusedInForeach++;
}

// But here it works:
$unusedIsOkay = 0;
$unusedIsOkay++;
```

**Comment by kalessil** [comment]
Fixed, new binary pushed

**Comment by kalessil** [comment]
Known false-positive: `return ++$whatever`, will fix it tomorrow

**Comment by kalessil** [comment]
Fixed =)
